alphagov/paas-admin
# Stub data api for /performance page

What
----

Page expects data to come from a `metrics.json` file. In staging and prod this is served by the  paas-admin-data` internal app, but for local development there was none.

This introduces an endpoint as part of the `stub-apis-env` script on `npm run start:stub-api` command.

Data is pseudo-randomly generated for the last year in weekly increments, much like the real data.

How to review
-------------

- clone branch
- run `npm install`
- run `npm run start:stub-api` to start stub apis
- run `npm run start:with-stub-api` to start the app
- visit http://localhost:3000/performance and check page has data
- review code

Who can review
---------------

not @kr8n3r 

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
